### ISSUE TITLE
REQUEST: reactions to the supports node number

### ISSUE THREAD

**Issue by Sicambre158** [karamba3d/K3D_NightlyBuilds#12]
Would it be possible to display at the output of the reactions to the supports component the node number to which each reaction or moment to the supports is attached instead of or in addition to the coordinates as is currently the case.

![image](https://user-images.githubusercontent.com/9198859/97779302-6ee07d80-1b7d-11eb-9232-b3fb3445c12d.png)

[https://grasshopperdocs.com/components/karamba3d/reactionForces.html](url)

**Comment by Clemens-Preisinger** [comment]
Yes, it would be possible. The question is what would be gained by adding the corresponding output-plug to the 'Reaction Forces'-component. For placing a support the node-coordinates as they are currently output are sufficient. The node index is something which may change from model to model so reusing the node-index from one model to construct a new model is risky. 
What advantaqes would you see?
-- Clemens

**Comment by Charles-Rhinoforyou** [comment]
The interest is to be able to make a request to know the reaction to the supports in a precise node in a single operation. Today it is necessary to recover the coordinates of the node then to make a proximity test to know what is the reaction to the supports at the nearest point
